public/js/UI.js

```diff
@@ -0,0 +1,129 @@
+var horoscope = {};
+
+$(document).ready(function() {
+    if ($('li').length > $('li:visible').length) {
+        $("#showMore").show();
+    } else {
+        $("#showMore").hide()
+    }
+    horoscope = new Horoscope({
+        //inializes horoscope object
+        sentence_types : {},
+        grammar : {},
+        sentenceBigramProbabilities : {},
+        sentenceTypes : {},
+        calendar : {},
+        userData : {
+            "name" : "",
+            "hometown" : "",
+            "birthday" : "",
+            "sign" : ""
+        },
+        date: "",
+        sentence : {
+            complete: false,
+            compound: false,
+            content: [],
+            tags: {},
+            possibleConversions : [],
+            testForAgreement : true,
+            newText : "",
+            cleanedContent : ""
+        } ,
+        structure : [],
+        paragraph : "",
+        loadGrammar : function(){
+            var that = this;
+            $.ajax({
+                url: "/json/grammar.json",
+                dataType: "json",
+                success: function(data) {
+                    that.grammar = data;
+                }
+            });
+        },
+        loadSentenceTypes : function(){
+          var that = this;
+          $.ajax({
+              url: "/json/sentenceTypes.json",
+              dataType: "json",
+              success: function(data) {
```

Comment:
aaahhhhhhh yes that should be a thing, although it looks like we're getting rid of Ajax in favor of an API